### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by Vishnu-Aithal**
Updated Readme

Preview - https://github.com/Vishnu-Aithal/Some-UI-Documentaion/tree/update-readme#readme

### CODE REVIEW

**Review by anik31 — APPROVED** [review]
Looks good.

Suggestion - put the live link at the top & add a screenshot or gif of the application to make it better

File: README.md [hunk 1]
```diff
@@ -1,2 +1,63 @@
...
+This is a Documentation Site of the SOME_UI Component Library - https://github.com/Vishnu-Aithal/Component-Library
+
+---
+
+> ### Quick Installation:
+
+Some_UI is an open source devoloping Component Library. To use this component library you have to link the stylesheet given below from cdn.
+Javascript Code Snippets are provided in the Documentation wherever neccessary.
+
+```
```
Comment (sainath7878) [inline README.md:14]: add html after the backticks so that the HTML tags gets highlighted

**Review by sainath7878 — APPROVED** [review]
Make the suggested changes and you're good to go. BTW, awesome UI.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,66 @@
...
+- Avatar
+- Badge
+- Cards
+- Images
+- Inputs
+- Lists
+- Navigation
+- Ratings
+- Toasts
+- Utility Classes
```
Comment (vrinda-mahajan) [inline README.md:63]: Suggestion- you can turn them into links.
Like this:
- [Buttons](https://some-ui.netlify.app/pages/documentation.html#buttons)

**Review by vrinda-mahajan — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Vishnu-Aithal) [reply README.md:14]: ok. Thanks for reviewing

File: README.md (hunk 2 above)
Comment (Vishnu-Aithal) [reply README.md:63]: ok. Thanks for reviewing

**Pull request merged**